src/api/app/controllers/webui/user_controller.rb

```diff
@@ -116,9 +116,9 @@ def register
       redirect_to controller: :user, action: :index
     else
       session[:login] = opts[:login]
-      User.session = User.find_by_login(session[:login])
-      if User.current.home_project
-        redirect_to project_show_path(User.current.home_project)
+      User.session = User.find_by!(login: session[:login])
```

Comment:
[nitpick] I agree with this change but is not in the correct commit

Reply:
you mean I should cherry pick in an extra commit? I squashes my original commits as it became a mess rebasing, but I can cherry pick special cases.

Reply:
I think she wants that change as extra commit, since you are not deprecating the ```User.current``` here

Reply:
yeah - clarified offline. It's too much hassle to seperate though, so we stay on nitpick